index.js

```diff
@@ -0,0 +1,92 @@
+const inputPosts = document.querySelector("#search-post")
+const getPost = document.querySelector(".getPost")
+const getComments = document.querySelector(".getComments")
+const body = document.querySelector("body")
+
+
+function getPostById(PostId) {
+  return new Promise((resolve, reject) => {
+    fetch(`https://jsonplaceholder.typicode.com/posts/${PostId}`)
+    .then((response) => {
+      if (response.status === 404) {
+        reject("404, Post not found")
+      }
+      return response.json()
+    })
+    .then((json) => {
+      return resolve(json)
+    })
+    .catch((err) => {
+      reject(err);
+    })
+  })
+}
+
+
+function getCommentByPostId() {
+  return new Promise((resolve, reject) => {
+    fetch(`https://jsonplaceholder.typicode.com/comments`)
```

Comment:
```suggestion
function getCommentByPostId(postId) {
  return new Promise((resolve, reject) => {
    fetch(`https://jsonplaceholder.typicode.com/comments?postId=${postId}`)
```

тут краще б було використати URL параметр, тоді Ви би завантажували значно менший обєм даних та нижче Вам не прийшлося би фільтрувати масив, щоб знайти необхідні коменти.
але ми це ще не розглядали